### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by RahullRawat**
image-input

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+:root {
+	--error-color: #dc3545;
+	--success-color: #1b7052;
+}
```
Comment (shaik-saad) [inline components/input/input.css]: You can add these variables in a seperate base file.

File: components/image/image.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+.image-round {
+	width: 25rem;
+	height: 25rem;
+	border-radius: 50%;
+	object-fit: cover;
+}
+
+.mt-4 {
+	margin-top: 4rem;
+}
```
Comment (shaik-saad) [inline components/image/image.css]: You can add this utility class in another separate file. Might help you in long run.

**Review by shaik-saad — APPROVED** [review]
Added some suggestions, other than that very clean code, great work Rahul! Also you're good to merge.

File: components/input/input.css (hunk 1 above)
Comment (RahullRawat) [reply components/input/input.css]: Done all the changes you have suugested. Thank you for suggestion

**Review by Kevin-Solomon — APPROVED** [review]

**Pull request merged**